hack/testing/logging.sh

```diff
@@ -438,6 +454,10 @@ else
     done
 fi
 
+if [ "$TEST_MIGRATE_UPGRADE" = "false" ] ; then
```

Comment:
this overlaps and is probably of no use with https://github.com/openshift/origin-aggregated-logging/pull/162